error-handling/src/test/java/com/palantir/remoting2/errors/SerializableErrorErrorDecoderTests.java

```diff
@@ -44,7 +44,7 @@
 
     private static final String message = "hello";
     private static final int STATUS_42 = 42;
-    private static final ObjectMapper OBJECT_MAPPER = ObjectMappers.newClientObjectMapper();
+    private static final ObjectMapper OBJECT_MAPPER = ObjectMappers.newServerObjectMapper();
```

Comment:
would be more realistic to serialize using the server object mapper and deserialize using the client one